src/catarse-admin.js

```diff
@@ -1,4 +1,5 @@
 var adminApp = window.adminApp = {
   models:{},
-  error: m.prop()
+  error: m.prop(),
+  timezone: 'America/Sao_Paulo'
```

Comment:
Let's not use timezone for now. I believe it's better to just adjust it in the server later on.